v2/internal/pkg/release/cincinnati.go

```diff
@@ -233,7 +236,7 @@ func (o *CincinnatiSchema) GetReleaseReferenceImages(ctx context.Context) []v2al
 }
 
 // getDownloads will prepare the downloads map for mirroring
-func getChannelDownloads(ctx context.Context, log clog.PluggableLoggerInterface, c Client, lastChannels []v2alpha1.ReleaseChannel, channel v2alpha1.ReleaseChannel, arch string) ([]v2alpha1.CopyImageSchema, error) {
+func getChannelDownloads(ctx context.Context, log clog.PluggableLoggerInterface, c Client, lastChannels []v2alpha1.ReleaseChannel, channel v2alpha1.ReleaseChannel, arch, mirrorMode, graphDataDir string) ([]v2alpha1.CopyImageSchema, error) {
```

Comment:
@aguidirh - also a nit: usually the standard for parameters length is 7 - we now have 8 (clean code - maintainability)